### PR TITLE
Test update with INIT_FILES and FORWARD_INIT

### PR DESCRIPTION
**Pull request by dafeda**
## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

File: test-data/local/externally_sampled_params/config.ert [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+QUEUE_OPTION LOCAL MAX_RUNNING 50
+
+RUNPATH out/realization-%d/iter-%d
+
+OBS_CONFIG observations
+TIME_MAP time_map
+
+NUM_REALIZATIONS 10
+
+GEN_KW COEFFS coeff.tmpl params.json coeff_priors FORWARD_INIT:True INIT_FILES:externally_sampled_params0
```
Comment (joakim-hove) [inline test-data/local/externally_sampled_params/config.ert:13]: I *think* the `INIT_FILES:/path/to/init/files%d` needs to contain a `%d` placeholder in order to ensure that the different realizations pick up different init files.

Comment (dafeda) [reply test-data/local/externally_sampled_params/config.ert:13]: My reading of [the docs](https://ert.readthedocs.io/en/latest/reference/configuration/data_types.html?highlight=INIT_FILES#initialization-with-forward-init) is that "%d" is not needed when `FORWARD_INIT=True` which means that the forward model is what does the sampling.
Is this perhaps wrong?

Comment (joakim-hove) [reply test-data/local/externally_sampled_params/config.ert:13]: I am sorry for the misleading comment - you had it right. 

My only remaining comment would be to recommend removing the literal `0` from the INIT_FILES name - that fooled me into thinking it represented a (fixed) realization number.

Comment (dafeda) [reply test-data/local/externally_sampled_params/config.ert]: Thanks, I've pushed a fix.

**Review by oyvindeide — APPROVED** [review]

**Pull request merged**